docs/infrastructure/deployment-targets/reflected-targets/index.md

```diff
@@ -0,0 +1,88 @@
+
+The variables used are different for each supported cloud provider.
+
+### Azure
+
+To discover Azure cloud resources, Octopus uses an [Azure account](/docs/projects/variables/azure-account-variables.md) variable named **Octopus.Azure.Account**.
+
+### AWS
+
+To discover AWS cloud resources, Octopus uses an [AWS account](/docs/projects/variables/aws-account-variables.md) variable named **Octopus.AWS.Account**. TODO: Additional variables ??? can be used to authenticate using an EC2 IAM role.
+
+## Tag cloud resources
+
+Octopus looks for tags applied to cloud resources to discover and create deployment targets automatically for you.
+
+Tags are in the format `octopus-{scope}` and support the following for discovery. Note that only a single value is supported in tags at the moment.
+
+- `octopus-environment`: The name of the [environment](/docs/infrastructure/environments) the target can be used in during deployments. This tag is required and only deployments for the environment will discover the target.
+- `octopus-role`: The [role](/docs/infrastructure/deployment-targets#target-roles) that should be applied to the target. This tag is required and only deployments with a step that matches the role will discover the target.
+- `octopus-space`: The name of the [space](/docs/administration/spaces) the target can be used in. This tag is optional and if present only deployments within the matching space can discover the target.
+- `octopus-project`: The name of the [project](/docs/projects) the target will be discovered by. This tag is optional and if present only deployments for the matching project can discover the target.
+- `octopus-tenant`: The name of the [tenant](/docs/projects) the target can be discovered for. This tag is optional and if present only deployments for the matching tenant will discover the target.
+
+## Examples
+
+### ECS Cluster
+
+As an example, let's say you have an project in Octopus called _Pet Shop_ that deploys an application to an ECS cluster in a _Development_ environment using a role of _PetShopFrontEnd_ and this cluster is dynamically created as part of the deployment using a CloudFormation template.
+
+To use this ECS cluster previously in Octopus you might have had to put in a [manual intervention step](/docs/projects/built-in-step-templates/manual-intervention-and-approvals) and manually create the target, or add a [script step](/docs/infrastructure/deployment-targets/dynamic-infrastructure/new-octopustarget.md) with custom code to try and find and create the ECS cluster target. In addition, previously when this cluster was no longer needed you might have needed to either [run a script](/docs/infrastructure/deployment-targets/dynamic-infrastructure/remove-octopustarget.md) or manually remove the target in Octopus.
```

Comment:
Not sure if anyone ever added a manual intervention to go in and registered targets - but they would certinly have had to have manually registered the target - perhaps we can simplify the first part to just that?